src/interfaces/coral_web/src/components/FirstTurnSuggestions.tsx

```diff
@@ -0,0 +1,99 @@
+import { Transition } from '@headlessui/react';
+import React, { useMemo } from 'react';
+
+import ButtonGroup from '@/components/ButtonGroup';
+import {
+  TOOL_CALCULATOR_ID,
+  TOOL_INTERNET_SEARCH_ID,
+  TOOL_PYTHON_INTERPRETER_ID,
+} from '@/constants';
+import { useListTools } from '@/hooks/tools';
+import { useParamsStore } from '@/stores';
+import { ConfigurableParams } from '@/stores/slices/paramsSlice';
+import { cn } from '@/utils';
+
+type Prompt = {
+  label: string;
+  params: Partial<ConfigurableParams>;
+  message: string;
+};
+
+const SUGGESTED_PROMPTS: Prompt[] = [
+  {
+    label: 'Plot real estate data',
+    params: {
+      fileIds: [],
+      tools: [
+        { name: TOOL_PYTHON_INTERPRETER_ID },
+        { name: TOOL_CALCULATOR_ID },
+        { name: TOOL_INTERNET_SEARCH_ID },
+      ],
+    },
+    message:
+      'Plot the average 1 bedroom rental price in Jan 2024 for the 5 most expensive cities in north america',
+  },
+  {
+    label: 'Clean up data in Python',
+    params: { fileIds: [], tools: [] },
+    message: `I want to figure out how to remove nan values from my array. For example, My array looks something like this:
```

Comment:
```suggestion
    message: `I want to figure out how to remove nan values from my array. For example, my array looks something like this:
```

Reply:
ah i think this one is still in the same sentence 😅

Reply:
jk i misread the wrong way thank you!!